src/main/java/org/sagebionetworks/bridge/udd/helper/SesHelper.java

```diff
@@ -123,4 +178,54 @@ private void sendEmailToAccount(AppInfo appInfo, AccountInfo accountInfo, Body b
         LOG.info("Sent email to account " + accountInfo.getUserId() + " with SES message ID " +
                 sendEmailResult.getMessageId());
     }
+
+    private MimeMessage makeRawEmailMessage(AppInfo appInfo, AccountInfo accountInfo, String attachment) throws MessagingException {
+        // from address
+        String fromAddress = appInfo.getSupportEmail();
+
+        // to address
+        String destination = accountInfo.getEmailAddress();
+
+        // subject
+        String appName = appInfo.getName();
+        String subjectStr = String.format(SUBJECT_TEMPLATE, appName);
+
+        // create and send message
+        Session session = Session.getDefaultInstance(new Properties());
+        MimeMessage message = new MimeMessage(session);
+
+        message.setSubject(subjectStr, CHARSET_UTF_8);
+        message.setFrom(new InternetAddress(fromAddress));
+        message.setRecipient(javax.mail.Message.RecipientType.TO, new InternetAddress(destination));
+
+        MimeMultipart messageBody = new MimeMultipart(CONTENT_SUBTYPE_ALTERNATIVE);
+        MimeBodyPart wrap = new MimeBodyPart();
+
+        MimeBodyPart textPart = new MimeBodyPart();
+        textPart.setContent(ATTACHMENT_BODY_TEMPLATE_TEXT, CONTENT_TYPE);
+
+        MimeBodyPart htmlPart = new MimeBodyPart();
+        htmlPart.setContent(ATTACHMENT_BODY_TEMPLATE_HTML, CONTENT_TYPE);
+
+        messageBody.addBodyPart(textPart);
+        messageBody.addBodyPart(htmlPart);
+
+        wrap.setContent(messageBody);
+
+        MimeMultipart mimeMessage = new MimeMultipart(CONTENT_SUBTYPE_MIXED);
+
+        message.setContent(mimeMessage);
+
+        mimeMessage.addBodyPart(wrap);
+
+        // define attachment
+        MimeBodyPart mimeAttachment = new MimeBodyPart();
+        DataSource dataSource = new FileDataSource(attachment);
+        mimeAttachment.setDataHandler(new DataHandler(dataSource));
+        mimeAttachment.setFileName(dataSource.getName());
```

Comment:
Do we need to set mime type "text/csv"?

Reply:
Do you mean we will want to set it as type "application/zip"?